test/unit/spec/v2/view-builder/views/ChallengeWebauthnView_spec.js

```diff
@@ -11,8 +11,10 @@ import ChallengeWebauthnResponse
   from '../../../../../../playground/mocks/data/idp/idx/authenticator-verification-webauthn.json';
 
 describe('v2/view-builder/views/webauthn/ChallengeWebauthnView', function () {
+  let test;
```

Comment:
nit: Consider using the variable `testContext` here to avoid confusion with the often reserved word `test`.